Library/Homebrew/extend/ENV/std.rb

```diff
@@ -59,33 +52,17 @@ def setup_build_environment(formula = nil)
       gcc_formula = gcc_version_formula($&)
       append_path "PATH", gcc_formula.opt_bin.to_s
     end
+  end
+  alias_method :setup_generic_build_environment, :setup_build_environment
 
-    # Add lib and include etc. from the current macosxsdk to compiler flags:
-    macosxsdk MacOS.version
-
-    if MacOS::Xcode.without_clt?
-      append_path "PATH", "#{MacOS::Xcode.prefix}/usr/bin"
-      append_path "PATH", "#{MacOS::Xcode.toolchain_path}/usr/bin"
-    end
-
-    # Leopard's ld needs some convincing that it's building 64-bit
-    # See: https://github.com/mistydemeo/tigerbrew/issues/59
-    if MacOS.version == :leopard && MacOS.prefer_64_bit?
-      append "LDFLAGS", "-arch #{Hardware::CPU.arch_64_bit}"
-
-      # Many, many builds are broken thanks to Leopard's buggy ld.
-      # Our ld64 fixes many of those builds, though of course we can't
-      # depend on it already being installed to build itself.
-      ld64 if Formula["ld64"].installed?
-    end
+  def homebrew_extra_pkg_config_paths
+    []
   end
 
-  # @private
   def determine_pkg_config_libdir
     paths = []
     paths << "#{HOMEBREW_PREFIX}/lib/pkgconfig"
     paths << "#{HOMEBREW_PREFIX}/share/pkgconfig"
-    paths << "#{HOMEBREW_ENV_PATH}/pkgconfig/#{MacOS.version}"
```

Comment:
Of course with something like `paths += homebrew_extra_pkg_config_paths` instead of using `<<` but I guess that's clear. (Just wanted to mention it to be better safe than sorry.)